README.md

```diff
@@ -13,33 +13,41 @@
 
 Using a fixed version:
 
-* [unpkg.com/react@16.0.0/umd/react.production.min.js](/react@16.0.0/umd/react.production.min.js)
-* [unpkg.com/react-dom@16.0.0/umd/react-dom.production.min.js](/react-dom@16.0.0/umd/react-dom.production.min.js)
+* [unpkg.com/react@16.0.0/umd/react.production.min.js](https://unpkg.com/react@16.0.0/umd/react.production.min.js)
```

Comment:
Please leave the links in the README alone and isolate your changes to just adding this one feature. Thanks! :D